testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadOpsTestCase.java

```diff
@@ -94,6 +94,7 @@ public void testWriteAttribvuteWithReload() throws Exception {
         assertTrue(result.isIsOutcomeSuccess());
         value = (String) result.getResult();
         assertThat(value, is("false"));
+        cli.sendLine("/subsystem=logging:write-attribute(name=add-logging-api-dependencies, value=true)");
```

Comment:
idem

Reply:
to reset the configuration to what was here before the test